certified-mail-sender/src/main/scala/it/pagopa/interop/certifiedMailSender/JobExecution.scala

```diff
@@ -0,0 +1,71 @@
+package it.pagopa.interop.certifiedMailSender
+
+import cats.implicits.{catsSyntaxFlatMapOps, toFunctorOps, toTraverseOps}
+import com.typesafe.scalalogging.Logger
+import courier._
+import io.circe.generic.auto._
+import io.circe.jawn.parse
+import it.pagopa.interop.certifiedMailSender.model.InteropEnvelope
+import it.pagopa.interop.commons.queue.impl.SQSHandler
+import it.pagopa.interop.commons.utils.TypeConversions.EitherOps
+
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+import scala.util.Try
+
+final case class JobExecution(sqsHandler: SQSHandler) {
+
+  private val logger: Logger = Logger(this.getClass)
+
+  implicit val mailer: Mailer = Mailer(ApplicationConfiguration.serverAddress, ApplicationConfiguration.serverPort)
+    .auth(true)
+    .as(ApplicationConfiguration.smtpUser, ApplicationConfiguration.smtpPassword)
+    .ssl(true)()
+
+  def run()(implicit blockingEC: ExecutionContextExecutor): Future[Unit] = sqsHandler.processAllRawMessages(
+    ApplicationConfiguration.maxNumberOfMessages,
+    ApplicationConfiguration.visibilityTimeout
+  ) { (messages, receipts) =>
+    logger.info(s"Sending ${messages.size} messages")
+    sendMessages(messages) >> deleteMessages(receipts)
+  }
+
+  private def sendMessages(messages: List[String])(implicit ec: ExecutionContext): Future[Unit] =
+    Future.traverse(messages)(sendMail).void
+
+  private def deleteMessages(receiptHandles: List[String])(implicit ec: ExecutionContext): Future[Unit] =
+    Future.traverse(receiptHandles)(sqsHandler.deleteMessage).void
+
+  private def sendMail(message: String)(implicit ec: ExecutionContext): Future[Unit] = for {
```

Comment:
for consistency both `sendMail`, or none of them, should use implicit mailer
```suggestion
  private def sendMail(message: String)(implicit mailer: Mailer, ec: ExecutionContext): Future[Unit] = for {
```